contrib-03-AnalysisWithPython/Readme.md

```diff
@@ -0,0 +1,153 @@
+  local python distribution as clean as possible. It can be achieve trough
+  : `conda create -n NameOfYourVirtualEnv python=2.7 anaconda`.
+  To activate it, do `source activate NameOfYourVirtualEnv` and `source deactivate` to go back to normal. 
+- Install ROOT inside the conda environment : `conda install -c chrisburr root=6.06.02`
+- You will then need to activate the conda environment :
+    - `source activate root` to load the main conda environment
+    - `source activate NameOfYourVirtualEnv` if you made it yourself
+- A few packages are also needed and can be installed using pip, the python
+  package manager :
+    - Numpy (arrays and tools for scientific computing), Matplotlib (plotting
+      library), scikit-learn (Machine Learning in Python) : `pip install numpy matplotlib scikit-learn`
+    - The [root_pandas package](https://github.com/ibab/root_pandas) is based on   [root_numpy](https://github.com/rootpy/root_numpy) and allows you to load a ROOT
+      file and store the content in a pandas dataframe. Follow both links if
+      you want more information on how to load a ROOT file. To install it, do
+      `pip install root_pandas`
+    - In order to produce a jupyter notebook which is a way to keep both the
+      analysis code and its output in the same file, you can also do `pip install jupyter`. Then, to load a jupyter notebook, you can do
+      `jupyter notebook` and access the notebook in your browser.
+
+## Loading data
+
+Using the root_pandas library, we can load a ROOT file in a few simple steps :
+- choose the file you want and store the path as string : `myfile = "path/to/file.root"`
+- then select the branches you want by creating a list with the names of the
+  variables you want. The use of of the wildcard `*` is also supported. For
+  instance you can select `mybranches = ["B0_M","*_PT"]` to load the B0 mass
+  and the transverse momentum of each particle of your entry.
+- As the file will be loaded in the memory, you should limit to a certain
+  number of entries and set, for instance, `n_events = 10000`.
+- Finally, select the name of your tuple as `tuple_name = 'someDir/myTree`
+
+Then, you will be allowed to store your tree as a pandas dataframe using :
+```python
+   import root_pandas
+   df = root_pandas.read_root(myfile, columns=mybranches, stop = n_events, key= tuple_name)
+```
+## Creating and plotting histograms
+
+To produce a simple plot, let's say, the mass of the B0, you can simply do :
+```python
+   import numpy as np
+   custom_binning = np.linspace(5000,5500,100) #min, max and number of bins 
+   mass_B0 = df['B0_M'].plot.hist(bins=custom_binning, label = 'mass of the
+   B^{0}_{M} [MeV/c$^{2}$]'} #label of the plot can be written in LaTeX
+```
+
+Then, you just need to add `plt.show()` in your code to plot the histogram.
+A more detailed code is written in both `compare_mc_data.py` and
+`AnalysisWithPython.ipynb`.  
+
+
+## Cutting
```

Comment:
Cutting -> Selecting events

In principle, "cut" is jargon so we should try to avoid it as much as possible